src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,136 @@
+package com.wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity(name = "book")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name = "genre")
+    private String genre;
+
+    @Column(name = "author", nullable = false)
+    private String author;
+
+    @Column(name = "image", nullable = false)
+    private String image;
+
+    @Column(name = "title", nullable = false)
+    private String title;
+
+    @Column(name = "sub_title", nullable = false)
+    private String subTitle;
+
+    @Column(name = "publisher", nullable = false)
+    private String publisher;
+
+    @Column(name = "year", nullable = false)
+    private String year;
+
+    @Column(name = "pages", nullable = false)
+    private int pages;
+
+    @Column(name = "isbn", nullable = false)
+    private String isbn;
+
+    public Book() {
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public Book setId(int id) {
```

Comment:
All the setters should return `void` instead of the object.

Reply:
Solved, i changed setters methods from return object to void method